packages/lib/date-ranges.ts

```diff
@@ -171,7 +168,7 @@ export function buildDateRanges({
     }, [])
   );
   const OOOdates = outOfOffice
-    ? Object.keys(outOfOffice).map((outOfOffice) => processOOO(dayjs(outOfOffice), timeZone))
+    ? Object.keys(outOfOffice).map((outOfOffice) => processOOO(dayjs.utc(outOfOffice), timeZone))
```

Comment:
This is a NIT, as the correct response is provided either way; but no reason not to parse outOfOffice in UTC mode.